msteams-platform/concepts/resource-specific-consent.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Resource specific consent in Teams
+description: Describes resource specific consent to apps accessing Team's data and how to configure it.
+keywords: RSC resource specific consent
+---
+# Resource specific consent in Teams
+
+Resource-specific consent (RSC) in Microsoft Teams allows team owners to grant consent to apps to access their team's data and replaces the need for a global admin to approve the app with [tenant-wide admin permissions](~/azure/active-directory/manage-apps/grant-admin-consent).
```

Comment:
There is noun inconsistency (apps/app) in this sentence.

Reply:
done